Blaise-Shyaka/Free-mentors
# Bg fix review box size id#167949384

#### What does this PR do?

It fixes the width of the mentor review box and enables the user to write a review from any device responsively and conveniently.

#### Description of Task to be completed

1. I have added a submit button, allowing the user to submit their review.
2. I have set the height and width of the 'mentor review box', to make the UI more attractive and give the user enough space to type in.
3. I have added a placeholder text on the mentor profile, to replace empty spaces.
4. I have replaced a placeholder image with an actual picture on the mentor's profile

#### How should this be manually tested?

1. On viewMentor.html, check the presence of a submit button below the mentor review box.
2. Check whether the height and width of the mentor review box is not the default one.
3. Check whether name, age, education level and bio fields on the mentor profile are not empty.
4. Check the lack of placeholder image on the mentor's profile.

#### Any background context you want to provide?

Not applicable

#### What are the relevant pivotal tracker stories?

1. A user should be able to view pages across all devices [#167949384]

#### Screenshots (if appropriate)

![increased the height and width of the review box](https://user-images.githubusercontent.com/52489421/64170388-4ad9bd00-ce50-11e9-8ae2-132e5ae16450.JPG)
![mentor profile with name and age fields filled in](https://user-images.githubusercontent.com/52489421/64170390-4b725380-ce50-11e9-86fd-18707747dac6.JPG)
